docs/pipelines.md

```diff
@@ -465,6 +465,22 @@ There are a lot of scenarios where `WhenExpressions` can be really useful. Some
 - Checking if the name of a CI job matches
 - Checking if an optional Workspace has been provided
 
+#### Guarding a `Task` and its dependent `Tasks`
+
+When  `WhenExpressions` evaluate to `False`, the `Task` and its branch (of dependent `Tasks`) will be skipped by
+default while the rest of the `Pipeline` will execute. The global default scope of `WhenExpressions` is set to `Branch`;
```

Comment:
i think that this deserves a bit more explanation as well - i.e. some explanation of "branch scope"